include/boost/math/tools/simple_continued_fraction.hpp

```diff
@@ -29,20 +29,28 @@
 
 namespace boost::math::tools {
 
-template<typename Real, typename Z = int64_t>
+template<typename Real, typename I = int64_t>
```

Comment:
The set of all integers is represented with Z: https://en.wikipedia.org/wiki/Integer. You'll see it throughout the code base.

Reply:
OK, my motivation was to use the same identifier as is used in `boost::rational`, but OK, I will stick to the `Z`.